client/src/components/Home/Home.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import './style.scss'
```

Comment:
maybe better add ; at the end, as I suppose they should be placed

Reply:
Yes, I will add. Thank you for seeing.